add ios shortcut documentation and add a recipe from api with tags (#233)
norish-recipes/norish: README.md

- Mobile apps.
- Public shareable recipe links.

> - Add direclty the recipe from Youtube, Instagram, TikTok or other without leaving the app.

Reviewer: ```suggestion
- Directly add a recipe from any app without leaving the app.
```
